src/main/java/duke/commands/DeleteCommand.java

```diff
@@ -0,0 +1,45 @@
+package duke.commands;
+
+import duke.DukeException.DukeException;
+import duke.tasks.TaskList;
+import duke.tasks.Task;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The `DeleteCommand` class is used to delete a task from a task list and returns a list of messages indicating
+ * the success of the operation.
+ */
+public class DeleteCommand extends Command {
+    private final String details;
+    private final List<String> messages = new ArrayList<>();
+
+
+    public DeleteCommand(String details) {
+        this.details = details;
+    }
+
+    /**
+     * This Java function deletes a task from a task list and returns a list of messages indicating the success of
+     * the operation.
+     * 
+     * @param tasks A TaskList object that contains a list of tasks.
+     * @return List of Strings messages which will be printed to user later.
+     */
+    public List<String> execute(TaskList tasks) throws DukeException {
```

Comment:
Should you add @ Override since you are overriding the abstract method from Command.java